### DISCUSSION TITLE
Custom Test Results on TestRail reporting

### DISCUSSION THREAD

**Discussion by gnapenas** [DamianOsipiuk/testcafe-reporter-testrail#24] [Q&A]
Is there a way to add custom test results to the testrail reporting? Currently our runs just say Test PASSED when the test is successful. On Failures it says Test Failed, but outputs the console error. 

On successful test passes I would like to output information on the testruns.

**Comment by DamianOsipiuk** [comment]
Hi, what kind of information would you like to post?
If you want to output the console dump of the test run, then unfortunately it is not possible.
This is the interface that is accessible from the reporter side: https://testcafe.io/documentation/402790/reference/plugin-api/reporter#testruninfo-object

As you can see we can access errors or warnings, but not the console dump.
Unless you are thinking about something else.

**Reply by gnapenas** [reply]
Yeah I was thinking outputting console dump or any strings and variables we would like to capture during the testruns
